dc_make/uncertainty.py

```diff
@@ -18,15 +18,17 @@ class UncertaintyType(Enum):
   shape = 2
 
 class Uncertainty:
-  def __init__(self, name, processes=None, eras=None, channels=None, categories=None):
+  def __init__(self, name, processes=None, eras=None, channels=None, categories=None, hasMultipleValues=False):
     self.name = name
     self.processes = processes
     self.eras = eras
     self.channels = channels
     self.categories = categories
+    self.hasMultipleValues=hasMultipleValues
 
   def appliesTo(self, process, era, channel, category):
-    return  Uncertainty.hasMatch(process, self.processes) \
+    match_subprocess = any(map(lambda p: Uncertainty.hasMatch(p, self.processes), process.subprocesses)) if process.subprocesses else Uncertainty.hasMatch(process.name, self.processes)
```

Comment:
```suggestion
    match_subprocess = any(map(lambda p: Uncertainty.hasMatch(p, self.processes), process.subprocesses)) if process.subprocesses else False
```

Reply:
why??

Reply:
it is already part of the OR the line below. Or I misread something?

Reply:
yes, indeed it can be put as False